en-US/Punctuation.xml

```diff
@@ -625,19 +625,139 @@ PLAY [Gather and display facts for managed nodes] ****************************</
+         However, in some cases, such an approach might not work, where tables are complex and include a variety of content, such as lists of strings, variables, IP addresses, descriptions, or states (selected or cleared). 
+         In such cases, treat each cell as an individual entry, and either use or omit a closing period depending on whether that cell is a complete sentence.
+		</para>
+		<para>
+			Example of a complex table with punctuation:		
+		</para>
+		 <table>
+			<title></title>
+			 <tgroup cols="2" colsep="1" rowsep="1">
+				<colspec colname="c1"></colspec>
+				 <colspec colname="c2"></colspec>
+				 <thead>
+					<row>
+						<entry> Field </entry>
+						 <entry> Value </entry>
+
+					</row>
+
+				</thead>
+				 <tbody>
+					<row>
+						<entry> 
+						Question
+						 </entry>
+						<entry> 
+						Default domain name
+						 </entry>
+
+					</row>
+					<row>
+						<entry> 
+						Description
+						 </entry>
+						<entry> 
+						This domain name is used to set the hostname of the node.
+						 </entry>
+
+					</row>
+					<row>
+						<entry> 
+						Answer variable name
+						 </entry>
+						<entry> 
+						<varname>global_domain_name</varname>
+						 </entry>
+
+					</row>
+					<row>
+						<entry> 
+						Answer type
+						 </entry>
+						<entry> 
+						text
+						 </entry>
+
+					</row>
+					<row>
+						<entry> 
+						Required
+						 </entry>
+						<entry> 
+						<emphasis>(selected)</emphasis>
+						 </entry>
+
+					</row>
+					<row>
+						<entry> 
+						Minimum length
+						 </entry>
+						<entry> 
+						1
+						 </entry>
+
+					</row>
+					<row>
+						<entry> 
+						Maximum length
+						 </entry>
+						<entry> 
+						251
+						 </entry>
+
+					</row>
+					<row>
+						<entry> 
+						Default answer
+						 </entry>
+						<entry> 
+						<code>lab.example.com</code>
+						 </entry>
+
+					</row>
+				</tbody>
+			</tgroup>
+		</table>
+
+	</section>	
+
  	 <section id="referring">
-		<title>Referring to Punctuation Marks</title>
+		<title>Referring to Punctuation Marks and Special Characters</title>
 		<para>
-			To refer to a punctuation mark or special character, use its name alone if it is one of the following standard characters:
+            To refer to a punctuation mark or special character, use its name alone if it is one of the following standard characters:
 		</para>
 		<para>
 			. , : ; ' " ( ) [ ] { } ! ?
 		</para>
 		<para>
-			For all other characters, use the character name followed by the symbol in parentheses.  
+            For all other characters, use the character name followed by the symbol in parentheses.  
+            When referring to a symbol in parentheses, the symbol name comes after any noun that describes the symbol.			
```

Comment:
I'm not quite understanding this line. What does "any noun" refer to, words like "symbol" or "sign"? Is it describing a use case like this: the greater than (`<`) symbol. 

One issue here is that not all symbol names include a noun like symbol or sign, so then the guidance around this might be unclear. And personally, I don't think we need to add extra words describing a symbol if we don't need to. 

How about something like this: For special character names that include nouns such as "sign" or "symbol", provide the character name, the symbol in parentheses, and then the noun that describes the symbol: the greater than (`<`) symbol. For special character names that do not include such nouns, provide the character name and the symbol in parentheses: a forward slash (`/`). In such cases, it is permissible to end a sentence with an encoded symbol.